Sources/SwifterSwift/UIKit/UIImageExtensions.swift

```diff
@@ -195,26 +195,36 @@ public extension UIImage {
         return newImage
     }
 
+    #if !os(watchOS)
```

Comment:
I think the same concept form #733 should apply here, where the watch OS check should be inside the function so we don't simply drop support of it for watchOS.

Reply:
Good point @LucianoPAlmeida, so maybe I could update the next function too (`withBackgroundColor(_: UIColor) -> UIImage`) to be consistent, what do you think?